src/main/java/appeng/util/Platform.java

```diff
@@ -29,7 +29,6 @@
 import java.util.List;
 import java.util.Random;
 import java.util.WeakHashMap;
-
 import javax.annotation.Nonnull;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') 'javax.annotation.Nonnull' should be separated from previous imports. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.imports.ImportOrderCheck)